src/MoBi.Core/Serialization/ORM/ContextPersistor.cs

```diff
@@ -114,7 +114,7 @@ public void Save(IMoBiContext context)
       private void verifyFileNotReadOnly(string projectFilePath)
       {
          var fileInfo = new FileInfo(projectFilePath);
-         if (!fileInfo.IsReadOnly)
+         if (!fileInfo.Exists || !fileInfo.IsReadOnly)
```

Comment:
@Yuri05 A file that does not exist IS readonly. WTF

Reply:
I guess it's more a philosopical question :)